pkg/kubelet/apis/config/doc.go

```diff
@@ -15,5 +15,6 @@ limitations under the License.
 */
 
 // +k8s:deepcopy-gen=package
+// TODO: This file should also enable openapi-gen like the other packages
```

Comment:
why? this is not served by an apiserver.

Reply:
We might consider this a bug, but the tag is needed in order for API violations to be tracked by the `api/api-rules/violation_exceptions.list`,  which I think is something we want to do. If we get all types to be tracked by that file automatically, while not generating any extra code, fine by me but for now let's track bad stuff there for componentconfigs as well

Reply:
Oh really? non-blocker for me. How you like.

Reply:
Gonna open an issue later about this